Resources/Prototypes/Entities/Mobs/Cyborgs/borg_chassis.yml

```diff
@@ -118,6 +118,8 @@
     access: [["Engineering"], ["Command"], ["Research"]]
   - type: Inventory
     templateId: borgShort
+  - type: SiliconLawProvider
```

Comment:
Since these are upstream files, I will need you to mark these changes with # DeltaV, along with a brief explaination of why these changes are here.

Reply:
Fixed